update deps

gabriel-v opened the pull request:
Scylla v0.14 and all of the others

Added new dev dependency for running all the tests that were starting with "truncate table" serially instead of in parallel ( i got some errors until i figured out to run with --test-thread=1 )

gabriel-v committed:
update deps

gabriel-v committed:
serial tests

Jasperav commented on Cargo.toml:
```diff
@@ -19,21 +19,22 @@ categories = ["database"]
-proc-macro2 = "1"
-serde = { version = "1", features = ["derive"] }
-serde_json = "1"
+scylla = "0.14.0" # This crate is specifically build for this version
```
This specific version dependency is fine, for the others, please remove the patch version :) so x.y and not x.y.z

Jasperav reviewed (changes requested):
Thanks for the PR! If you made the changes to the dependency and bump the Cargo.toml, I will publish the new version.

gabriel-v committed:
remove patch versions from workspace

gabriel-v committed:
bump 0.2 & remove dead code "Cursor"

gabriel-v commented:
ok

bumped workspace version 0.2.0, there are breaking changes:
- `select_paged` has different argument type
- the "Cursor" pub typedef was not used anymore

Jasperav commented:
@gabriel-v LGTM, do you know why this error occurs on the CI? https://github.com/Jasperav/Catalytic/actions/runs/11437836428/job/31829718889?pr=48. If the CI is ok, I will publish the new version!

gabriel-v committed:
regenerate lockfile using stable compiler

gabriel-v committed:
cargo fmt

gabriel-v committed:
ignore some more clippy errors

gabriel-v committed:
mv: remove "is not null" for non-pk field (scylla 5.4, 6.*)

gabriel-v committed:
github action: use all available threads

gabriel-v committed:
update stable toolchain 1.81

gabriel-v commented:
here you go

https://github.com/gabriel-v/Catalytic/actions/runs/11445259484

errors in order:
- cargo lock version = 4 was from my local nightly compiler, when switched to stable, it now outputs version = 3
- muted some clippy errors about comment string formatting, unrelated to patch
- muted clippy error on dead code (Field::attribute, undocumented, for macro?) - plz review
- removed "... and type is not null" from materialized view example - this errors (as type is not PK) in scylla >5.4.X, 6.x but not scylla 5.2.X nor cassandra 3.11
- updated CI compiler toolchain to the newer one i had locally, since the compiler error text tests depend closely on compiler version (different structs implementing missing trait get printed)

Jasperav commented:
Thanks a lot @gabriel-v, I published the changes!
